src/main/resources/templates/user-transactions-template.html

```diff
@@ -106,7 +105,7 @@ <h2>Transactions</h2>
                     <th class="table-border-right">Pending</th>
                     <th>Paid Out</th>
                 </tr>
-                <tr th:each="tx,iStat : ${txSummary.transactions}" th:classappend="${(iStat.even? 'table-grey-background': '') + (iStat.last? 'table-border-bottom': '')}">
+                <tr th:each="tx,iStat : ${txSummary.transactions}" th:classappend="${(iStat.even? 'table-grey-background': '') + ' ' + (iStat.last? 'table-border-bottom': '')}">
```

Comment:
adding + ' ' + doesn't fix the issue table-border-bottom on last transaction

Reply:
Without space, the last even row wasn't gray.